react/src/components/organisms/TeaserListing/index.js

```diff
@@ -26,146 +23,103 @@ class TeaserListing extends React.Component {
-                   const key = `teaser-listing--item-${index}`;
-                   return(
-                     <li className="ma__teaser-listing__item" key={key}>
-                       <GeneralTeaser level={teaserHeading} {...item} />
-                     </li>
-                   );
-                 })}
-               </ul>
-               <Collapse in={this.state.open} dimension="height">
-                 <div className="ma__teaser-listing__extra">
-                   <ul className={itemsClasses}>
-                     {invisibleItems.map((item, index) => {
-                       const key = `hidden-teaser-listing--item-${index}`;
-                       return(
-                         <li className="ma__teaser-listing__item" key={key}>
-                           <GeneralTeaser level={teaserHeading} {...item} />
-                         </li>
-                       );
-                     })}
-                   </ul>
-                 </div>
-               </Collapse>
-               <button
-                 className="ma__content-link ma__content-link--chevron ma__content-link__acordion-toggle js-accordion-link"
-                 aria-label={(this.state.open) ? teaser.lessLabel : teaser.moreLabel}
-                 onClick={this.handleClick}
-               >
-                 {(this.state.open)
-                   ? (<span className="less">{teaser.lessLabel}</span>)
-                   : (<span className="more">{teaser.moreLabel}</span>)
-                 }
-               </button>
-             </React.Fragment>
-           )}
-           {(invisibleItems.length === 0) && (
-             <ul className={itemsClasses}>
-               {teaser.items.map((item, index) => {
-                 const key = `teaser-listing--item-${index}`;
-                 return(
-                   <li key={key} className="ma__teaser-listing__item">
-                     <GeneralTeaser {...item} />
-                   </li>
-                 );
-               })}
-             </ul>
-           )}
-           {(teaser.more) && (
-             <div className="ma__teaser-listing__more">
-               <Link {...teaser.more} />
-             </div>
-           )}
+           {children}
+
          </div>
        </section>
      );
    }
 }
 
+
 TeaserListing.propTypes = {
-  /** Optional CompHeading component */
-  compHeading: PropTypes.shape(CompHeading.propTypes),
-  /** Optional SidebarHeading component */
-  sidebarHeading: PropTypes.shape(SidebarHeading.propTypes),
-  /** Descriptive paragraph */
-  description: PropTypes.shape({
-    text: PropTypes.string
-  }),
+  /** React children to render */
+  children: PropTypes.node.isRequired
+};
+
+TeaserListing.displayName = 'TeaserListing';
+export default TeaserListing;
+
+/**
+  Feature Items wrapper
+  */
+
+const TeaserListingFeatures = ({ children, stacked }) => {
+  const featuredClasses = classNames({
+    'ma__teaser-listing__featured-items': true,
```

Comment:
Setting the string to true isn't needed here:
```suggestion
    'ma__teaser-listing__featured-items',
```